### PR TITLE
Update sharing-boards.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Revised steps to access share link.

### CODE REVIEW

File: source/boards/sharing-boards.rst [hunk 1]
```diff
@@ -21,7 +21,7 @@ Sharing Boards
 To share a board with others:
 
 1. Open a board in any view.
-2. On the top-right of the board select the options menu **...**, then select **Share board**.
+2. On the top-right of the board select **Share board**.
 3. Set the toggle on to **Publish to web and share this board with anyone**.
```
Comment (wuwinson) [inline source/boards/sharing-boards.rst:25]:
```suggestion
3. Copy the internal link to **Share an editable view with users who have permissions to the board**.
-- OR --
4. Switch to the Publish tab and set the toggle on to **Publish and share a "read-only" link with everyone on the web**.
```

File: source/boards/sharing-boards.rst [hunk 2]
```diff
@@ -21,7 +21,7 @@ Sharing Boards
...
 4. Copy the link provided.
```
Comment (wuwinson) [inline source/boards/sharing-boards.rst:26]:
```suggestion
5. Copy the link provided.
```

File: source/boards/sharing-boards.rst (hunk 1 above)
Comment (wuwinson) [reply source/boards/sharing-boards.rst:25]: There's an extra "internal link" option in the Share modal. Maybe using numbered bullet points isn't the best with the "OR" option, but added details on the new functionality. Feel free to fix up the formatting as you see fit @justinegeffen.

**Review by wuwinson — CHANGES_REQUESTED** [review]

File: source/boards/sharing-boards.rst [hunk 3]
```diff
@@ -21,8 +21,12 @@ Sharing Boards
 To share a board with others:
 
 1. Open a board in any view.
-2. On the top-right of the board select the options menu **...**, then select **Share board**.
-3. Set the toggle on to **Publish to web and share this board with anyone**.
+2. On the top-right of the board select **Share board**.
```
Comment (cwarnermm) [inline source/boards/sharing-boards.rst:24]:
```suggestion
2. Select **Share board** in the top-right corner of the board.
```

File: source/boards/sharing-boards.rst [hunk 4]
```diff
@@ -21,8 +21,12 @@ Sharing Boards
...
+3. The following two options are available:
+
+   * Copy the internal link to **Share an editable view with users who have permissions to the board**.
```
Comment (cwarnermm) [inline source/boards/sharing-boards.rst:27]:
```suggestion
   * On the **Share** tab, you can share an editable view link with users who have permissions to the board.
```

File: source/boards/sharing-boards.rst [hunk 5]
```diff
@@ -21,8 +21,12 @@ Sharing Boards
...
+   * Switch to the **Publish** tab and set the toggle on to **Publish and share a "read-only" link with everyone on the web**.
```
Comment (cwarnermm) [inline source/boards/sharing-boards.rst:28]:
```suggestion
   * On the **Publish** tab, you can share a read-only link online with everyone.
```

**Review by cwarnermm — APPROVED** [review]
I've provided a couple of suggestions to consider. Approving for expediency.

File: source/boards/sharing-boards.rst [hunk 6]
```diff
@@ -21,8 +21,12 @@ Sharing Boards
 To share a board with others:
 
 1. Open a board in any view.
-2. On the top-right of the board select the options menu **...**, then select **Share board**.
-3. Set the toggle on to **Publish to web and share this board with anyone**.
+2. Select **Share board** in the top-right corner of the board.
+3. The following two options are available:
+
+   * On the **Share** tab, you can share an editable view link with users who have permissions to the board.
+   * On the **Publish** tab, you can share a read-only link online with everyone.
```
Comment (chenilim) [inline source/boards/sharing-boards.rst:28]: The "Enable Publicly-Shared Boards" must be turned on in the System Console for this option to be enabled (it's currently off by default). Just checking that this is covered somewhere already. Suggest linking to it if so.

**Review by chenilim — APPROVED** [review]
Looks good, but we should document the System Console setting if not already and link to it.

**Review by wuwinson — APPROVED** [review]
Looks good to me! And good point @chenilim - I don't think we have any documentation on the "Enable Publicly-Shared Boards" setting, but we should get that in.

@justinegeffen below are the steps for your reference:

To enable the share button on Boards:
From the app switcher, go to System Console > Mattermost Boards (Under "Plugins") > set "Enable Publicly-Shared Boards" to "True"
Screen Shot 2022-03-14 at 11 40 15 PM

**Pull request merged**